benchmark/test_special_perf.py

```diff
@@ -140,6 +140,36 @@ def padding_kwargs(dtype, batch, size):
     bench.run()
 
 
+def test_perf_upsample_nearest2d():
+    def upsample_nearest2d_kwargs(dtype, batch, size):
+        channel = 16
+        scale_factors = (2, 2)
+        shape = (batch, channel, size, size)
+        output_size = (
+            int(shape[2] * scale_factors[0]),
+            int(shape[3] * scale_factors[1]),
+        )
+        input = torch.randn(size=shape, device="cuda", dtype=dtype)
+        return {
+            "input": input,
+            "output_size": output_size,
+            "scales_h": None,
+            "scales_w": None,
+        }
+
+    bench = Benchmark(
+        op_name="upsample_nearest2d",
+        # py api torch.nn.functional.interpolate,
+        torch_op=torch._C._nn.upsample_nearest2d,
```

Comment:
Can we just use the pytorch api torch.nn.functional.interpolate with proper args to make the api call the torch._C._nn._upsample_nearest2d functions